modules/tile-converter/src/i3s-converter/helpers/geometry-converter.ts

```diff
@@ -1384,6 +1387,20 @@ function generateAttributeBuffer(type: string, value: any): ArrayBuffer {
  */
 function getAttributeType(key: string, attributeStorageInfo: any[]): string {
   const attribute = attributeStorageInfo.find((attr) => attr.name === key);
+  if (!attribute) {
+    console.log(
```

Comment:
I would recommend using `@probe.gl/log` - it will deduplicate warnings so you don't get cascades etc. Not for this PR obv.